src/main.rs

```diff
@@ -1,5 +1,8 @@
 #![allow(clippy::redundant_closure)]
 
+#[macro_use]
+extern crate text_io;
```

Comment:
Nitpick: you can get rid of this extern crate + macro use and just do a ‘use text_io::read’

Reply:
`text_io` only exposes macros and if I use `text_io::read!` it `cannot find macro `try_read!` in this scope`.